package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "federation-rs",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
Is the introduction of this file intentional? And if so, why?

Reply:
I think I may have run the npm install from the wrong directory initially. Made this PR very late last night and logged off immediately, thanks for the catch.

Reply:
Ah, that checks out.  Thanks!  Approved!